tests/005_define.test

```diff
@@ -18,7 +26,7 @@ meta-product = \${TEST_DEFINE}
 # Test substitution in a resource name
 file-resource "\${TEST_DEFINE}" {
         # Test substitution in the middle of a string
-        host-path = "$TESTS_DIR/\${TEST_DEFINE}"
+        host-path = "$TEST_PATH"
```

Comment:
This change removes one of the things that the unit test is testing. See the comment about testing substitutions in the middle of strings on the line right above.